tests/CommandRunnerTest.cpp

```diff
@@ -34,7 +34,7 @@ TEST(CommandRunnerTest, should_force_SIGKILL_inifinite_loop_command) {
 }
 
 TEST(CommandRunnerTest, should_not_crash_when_child_throws) {
```

Comment:
@kamaradclimber , I forgot you were still working remotely. So, you can then handle the patch if you wish and you have time.

After second look, you should probably keep this one but augment it checking no error is returned. It will cover the branch with `break`.
For the second branch, we can add a new test checking an error is returned and we can also check its content.

Reply:
done and I've fixed a small bug (we were not displaying the real exit code but something else)